.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit =
+    tests/*
```

Comment:
Should this be `unit-tests/*` instead of `tests/*` since our tests are in the unit-tests/ directory?

Reply:
I will fix that. Thank you!